runtime/src/iree/base/internal/BUILD

```diff
@@ -146,6 +146,7 @@ iree_runtime_cc_library(
 iree_runtime_cc_test(
     name = "file_io_test",
     srcs = ["file_io_test.cc"],
+    tags = ["no-qemu"],
```

Comment:
This also won't run on the web. Unless there's a more canonical tag, how about "requires-filesystem"?

Reply:
Done.